README.md

```diff
@@ -35,7 +35,7 @@ urlwatch 2 requires:
 
 The dependencies can be installed with (add `--user` to install to `$HOME`):
 
-`python3 -m pip install pyyaml minidb requests keyring appdirs lxml cssselect`
+`python3 -m pip install pyyaml minidb requests keyring appdirs lxml cssselect beautifulsoup4 jsbeautifier cssbeautifier`
```

Comment:
Can/should we make those optional, so that we don't pull in all those modules by default?
Import them only inside the `filter()` method of the beautify class.

See also "Optional dependencies" below.